app/jobs/scheduled/aggregate_web_hooks_events.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Jobs
+  class AggregateWebHooksEvents < ::Jobs::Scheduled
+    every 1.day
+
+    def execute(args)
+      today_web_hooks_events =
+        WebHookEvent.where(created_at: Time.zone.now.beginning_of_day..Time.zone.now.end_of_day)
+      today_web_hooks_events.each do |web_hooks_event|
```

Comment:
This isn't right -- you are making a `WebHookEventsDailyAggregate` record for every single `WebHookEvent` in a day -- you should only be making one for every `WebHook` that had 1 or more events fire that day.